CONTRIBUTING.rst

```diff
@@ -61,8 +61,8 @@ of the core developers before it is merged into the main repository.
 #. If you feel uncomfortable or uncertain about an issue or your changes, feel
    free to email the *sphinx-dev* mailing list.
 #. Fork `the repository`_ on GitHub to start making your changes to the
-   **master** branch for next major version, or **stable** branch for next
-   minor version.
+   **master** branch for next major version, or **X.Y** branch for next
```

Comment:
One more nit: I see you use bold (\*\*) here and literals (\`\`) later on. Could you use literals here too?